### DISCUSSION TITLE
Extra Water Temp Probe with EasyTouch

### DISCUSSION THREAD

**Discussion by Telemachus1977** [tagyoureit/nodejs-poolController#627] [App and O/S Config Questions]
When my friend's pool was built a few years back, they plumbed in the water temperature probe AFTER the heater... which means the feedback loop for both the gas and solar heater were always wonky.  He remedied this by inserting a new 10k probe into a more appropriate place pre heater and connected the new probe to his Easy Touch 4.

Now that he has NJS-PC and REM running alongside his ET, we were able to connect the original probe up to a cheap ADC card.  I had to "reverse bias" to get meaningful results, which I assume means I wired the cables in reverse?

![Screenshot_20221106_203328_Chrome](https://user-images.githubusercontent.com/25725743/200231282-fde2730a-bbb6-4bfb-8956-1d4985b61163.jpg)

Anyways, this is slightly useful information for pool nerds because this temperature probe will now give the differential between water going into the heater and coming out.

Obviously the Easy Touch system does not have any way to interpret this temperature.

Is there any way that it could be displayed on the homepage of njs-pc-dp?

**Comment by tagyoureit** [comment]
The 10k probes are just resistors... they can work in either direction.  It's probably the way the card reads the signal that needed to be reversed.  Did you see https://github.com/tagyoureit/nodejs-poolController/discussions/424?  

I have on my Wishlist to add custom attributes/cards to the dP pages, but am not there yet.

**Reply by Telemachus1977** [reply]
Thanks, interesting discussion on that board.  I need to dig into it some more.

On the Nixie side it allows up to 3 solar temps...  Could I add another solar heater and feed the temp to that so it will display (but not use that heater obviously)? Or does the fact that is connected to Easy Touch make all of that undoable?

**Reply by tagyoureit** [reply]
We don't have the ability yet to display random/additional measurements. It's on the wish list. 

To get it to display the solar work EasyTouch you would need to enable solar which you don't want to do.
